daemon/dns/dns_resolvconf_file.go

```diff
@@ -59,7 +63,7 @@ func resetDNSinResolvconfFile(addresses []string) error {
 	// set DNS
 	_ = internal.FileUnlock(resolvconfFilePath)
 	defer internal.FileLock(resolvconfFilePath)
-	content := "# Generated by NordVPN\n" + strings.Join(addrs, "\n")
+	content := resolvconfFileMark + "\n" + strings.Join(addrs, "\n")
```

Comment:
[small] might be nice to add a `\n` after the servers, so `cat /ect/resolv.conf` moves the prompt on new line? :smile: